balances-tx-payment/rpc/Cargo.toml

```diff
@@ -20,17 +20,20 @@ parking_lot = "0.12.0"
 thiserror = "1.0.31"
 
 # local deps
-encointer-balances-tx-payment-rpc-runtime-api = { package = "encointer-balances-tx-payment-rpc-runtime-api", path = "runtime-api", version = "2.0.0" }
-encointer-primitives = { path = "../../primitives", version = "2.0.0" }
-encointer-rpc = { path = "../../rpc", version = "2.0.0" }
+encointer-balances-tx-payment-rpc-runtime-api = { package = "encointer-balances-tx-payment-rpc-runtime-api", path = "runtime-api", version = "3.0.1" }
+encointer-primitives = { path = "../../primitives", version = "3.0.1" }
+encointer-rpc = { path = "../../rpc", version = "3.0.1" }
 
 # substrate deps
-pallet-transaction-payment = { default-features = false, version = "24.0.0" }
-pallet-transaction-payment-rpc = { default-features = false, version = "26.0.0" }
-sc-rpc = "25.0.0"
-sc-rpc-api = "0.29.0"
-sp-api = "22.0.0"
-sp-blockchain = "24.0.0"
-sp-core = "24.0.0"
-sp-rpc = "22.0.0"
-sp-runtime = "27.0.0"
+pallet-transaction-payment = { default-features = false, version = "25.0.0" }
+pallet-transaction-payment-rpc = { default-features = false, version = "27.0.0" }
+sc-rpc = "26.0.0"
+sc-rpc-api = "0.30.0"
+sp-api = "23.0.0"
+sp-blockchain = "25.0.0"
+sp-core = "25.0.0"
+sp-rpc = "23.0.0"
+sp-runtime = "28.0.0"
+
+[features]
+std = []
```

Comment:
Was this added by zepter? Not sure if I agree that it should be here, when a crate can't compile to `no_std`.

Reply:
yes, probably zepter added it because there was the above mentioned no-default-features present